web/src/inbox_util.js

```diff
@@ -0,0 +1,59 @@
+import $ from "jquery";
+
+import * as compose_state from "./compose_state";
+import * as overlays from "./overlays";
+import * as popovers from "./popovers";
+import * as stream_color from "./stream_color";
+import * as stream_data from "./stream_data";
+
+let is_inbox_visible = false;
+
+export function set_visible(value) {
+    is_inbox_visible = value;
+}
+
+export function is_visible() {
+    return is_inbox_visible;
+}
+
+export function get_pm_key(msg) {
+    return "pm:" + msg.other_user_id;
```

Comment:
We should probably rename these to `dm:` for this PR just to save a bit of migration effort. (I think we already did that migration for Recent Conversations, though `filter_pm` is a thing that still exists there and should be changed when it's convenient, with care taken around the local storage migration issue, if any).

Reply:
Renamed pm to dm.